src/pemfile.rs

```diff
@@ -2,7 +2,7 @@ use std::io::{self, ErrorKind};
 
 /// The contents of a single recognised block in a PEM file.
 #[non_exhaustive]
-#[derive(Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
We usually prefer to avoid `Clone` implementations on private key material. Why do you end up needing it here?

Reply:
This was to allow `self` to be consumed, while allowing the `Item` to be retained, if needed. I discussed a few alternatives in the original comment. Open to other approaches as well—those are just what came to mind.

Reply:
Maybe you're right. When I was using the crate, it just felt incongruous that there was the `certs` utility method, but no equivalent for keys.

Reply:
> so maybe this API isn't worth it?

It seems like changing the work to avoid `Clone` makes the value of the change fairly marginal, while keeping `Clone` on `Item` doesn't sound acceptable. 

Without other alternatives it feels like we're at an impasse here and I'm inclined to close the PR rather than let it languish without a path forward. What do you folks think?

Reply:
Agreed.